pkg/virt-operator/resource/generate/components/validations_generated.go

```diff
@@ -8070,6 +8070,10 @@ var CRDsValidation map[string]string = map[string]string{
                 type: string
             type: object
           type: array
+        isoAlignmentMode:
```

Comment:
What if we would start recording the KubeVirt version which was used to create  the VM? It would allow us to do compatibility decisions without having to always introduce new statuses. I think that would be cleaner on the long run.